### ISSUE TITLE
[pixi] String escaping doesn't work correctly

### ISSUE THREAD

**Issue by nainar** [ampproject/amp.dev#4522]
See below screenshot

![Screen Shot 2020-09-14 at 12 11 58 PM](https://user-images.githubusercontent.com/7919520/93127930-99d14980-f683-11ea-8c73-8a5699ff486d.png)

**Comment by nainar** [comment]
cc @kristoferbaxter @caroqliu

**Comment by tharders** [comment]
The problem with the escaped html entities in the recommendation text and status banner are related to this bug: https://github.com/ampproject/amphtml/issues/27153

**Comment by tharders** [comment]
The provided fix keeps some problem open.
We still need to convert markdown for the text we get from Lighthouse.
The problem there is that we cannot even ensure that the text does not contain certain characters.
The other problem is that there is another [issue](https://github.com/ampproject/amphtml/issues/30273) with amp-script and innerHTML when we use `<pre>` tags which we get when we have code examples in the recommendations.

I have created a PR to work around these problems at the pixi side, but I am not very happy with it.
I had to replace the angle brackets with similar unicode characters in order to display them. 
The problem with that is the user cannot copy a working piece of code.
And it is still possible to create markup that would result in display errors like code examples with highlights.
